Atomic/util.py

```diff
@@ -254,3 +253,19 @@ def urllib3_disable_warnings():
             # Except only call disable-warnings if it exists
             if hasattr(urllib3, 'disable_warnings'):
                 urllib3.disable_warnings()
+
+
+def skopeo(image):
```

Comment:
`myregistry.com:5000` and `docker.io` (in this order) are returned by the `/info` endpoint and before calling `skopeo` we should fully qualify the _unqualified_ image got from the cli (if it is unqualified, otherwise let's just call `skopeo` with the `image` name provided by the command line)